instagram/src/components/PostContainer/PostHeader.js

```diff
@@ -1,11 +1,13 @@
 import React from 'react';
```

Comment:
nicely done import of reusable styles, hope you can get some more styling in, just realized you didn't style much and your app had default styling. Maybe due to the uniqueness, but i kind of like this as well.